mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -556,11 +556,38 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 						"enable_confidential_compute": {
 							Type:        schema.TypeBool,
 							Required:    true,
+							ForceNew:    true,
 							Description: `Defines whether the instance should have confidential compute enabled.`,
 						},
 					},
 				},
 			},
+			"advanced_machine_features": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Computed:    true,
```

Comment:
@upodroid: Just curious, why did you choose to make the value `Computed: true`? I haven't seen the API return a default value, does it have a default that changes based on other configuration between the resource? (sorry for the drive-by comments here, there's a conversation internally about the implementation of the field)

Reply:
Values were computed in `google_compute_instance` so I copied them over. I did the bulk of the testing with `google_compute_instance`

Reply:
Thanks! And GCE Instance will return the value when not sent in some circumstances?

Reply:
Also see https://github.com/hashicorp/terraform-provider-google/issues/9436 if you're interested in some of the context here, I'm seeing @karlkfi decided to comment around the same time I did 🙂